lmdeploy/pytorch_poc/utils.py

```diff
@@ -5,12 +5,17 @@
 from typing import Dict, Sequence
 
 import psutil
-import torch
+import pycuda.driver as drv
```

Comment:
may update requirements.txt for `pycuda`?

Reply:
done.